homebrew_challenge/src/App.jsx

```diff
@@ -451,86 +443,13 @@ const App = () => {
-            <Col xs={12} md={4} className="p-3 d-flex justify-content-around">
-              <a
-                href="https://www.gov.scot/collections/coronavirus-covid-19-guidance/"
-                target="_blank"
-                rel="noopener noreferrer"
-                className="link"
-              >
-                See the latest Scottish Government Covid-19 guidance
-              </a>
-            </Col>
-            <Col xs={12} md={4} className="p-2 d-flex justify-content-around">
-              <a
-                href="https://www.scottishtecharmy.org/"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <img
-                  src="STABanner.png"
-                  alt="Scottish Tech Army"
-                  width="270"
-                  height="50"
-                />
-              </a>
-            </Col>
-            <Col xs={12} md={4} className="p-3 d-flex justify-content-around">
-              <a
-                href="https://www.nhsinform.scot/illnesses-and-conditions/infections-and-poisoning/coronavirus-covid-19"
-                target="_blank"
-                rel="noopener noreferrer"
-                className="link"
-              >
-                For Covid-19 health information visit NHS Inform
-              </a>
-            </Col>
-          </Row>
-          <Row>
-            <Col xs={12}>
-              <div className="sitemap p-2">
-                <div className="title">Sitemap</div>
-                {sitemapEntry(PAGE_PUBLIC_DASHBOARD, "Public Dashboard")}
-                {sitemapEntry(PAGE_ANALYTICS_DASHBOARD, "Analytics Dashboard")}
-                {sitemapEntry(
-                  PAGE_DATA_SOURCES,
-                  "Data sources and attribution"
-                )}
-                {sitemapEntry(PAGE_ABOUT_US, "About us")}
-              </div>
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <div className="footer-copyright py-3">
-                Unless otherwise stated, this webpage contains public sector
-                information licensed under{" "}
-                <a
-                  href="http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
-                  target="_blank"
-                  rel="noopener noreferrer licence"
-                  className="link"
-                >
-                  the Open Government Licence 3.0.
-                </a>
-                <br />© 2020 Copyright:&nbsp;
-                <a
-                  href="https://www.scottishtecharmy.org/"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="link"
-                >
-                  ScottishTechArmy.org
-                </a>
-              </div>
-            </Col>
-          </Row>
-        </Container>
-      </footer>
+      <Footer
+        PAGE_PUBLIC_DASHBOARD = {PAGE_PUBLIC_DASHBOARD}
```

Comment:
I wouldn't pass in constants like this (as props). If there are 2 components sharing constants, create a consts file with the constants and have both components import. eg DataChartsConsts.js

Reply:
done